Tests/ArcGISToolkitTests/FloorFilterViewControllerTests.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import ArcGISToolkit
+import ArcGIS
+
+class FloorFilterViewControllerTests: XCTestCase {
+    func testTopPlacement() {
+        let floorFilterView = FloorFilterViewController.makeFloorFilterView(geoView: AGSGeoView(frame: .zero), xMargin: UIScreen.main.bounds.width - 100, yMargin: 100)
+        let xPositionOfFloorFilterView = UIScreen.main.bounds.width - CGFloat(100) - CGFloat(floorFilterView?.view.bounds.width ?? 0)
+        XCTAssertEqual(UIScreen.main.bounds.width - 100 - 50, xPositionOfFloorFilterView)
```

Comment:
Because this test relies on the bounds of the main screen, it is device dependent. Also, this is a [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions), which is generally to be avoided.

Same with other tests.

Reply:
That makes sense. I have removed the tests for the placements. Also regarding the Yoga condition, my other tests have the` XCTAssert(actualValue, expectedValue)`. I thought that in the Yoda condition, it is the opposite? Let me know if I am missing some in the XCTAssert lines.